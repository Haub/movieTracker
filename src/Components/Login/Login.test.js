import React from 'react';
import { Login } from './index.js';
import { shallow } from 'enzyme';

describe('LOGIN', () => {
let wrapper;
  
  beforeEach(() => {
    const wrapper = shallow(<Login />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have an initial state with name, email, password, signup, avatar, and error', () => {
    const wrapper = shallow(<Login />);
    const expected = {"avatar": "", "email": "", "error": "", "name": "", "password": "", "signUp": false}
    expect(wrapper.state()).toEqual(expected);
  });

})