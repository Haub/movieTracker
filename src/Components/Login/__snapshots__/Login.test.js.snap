// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOGIN should match the snapshot 1`] = `undefined`;
